lib/client.js

```diff
@@ -75,8 +82,22 @@ function messageHandler(client, event) {
       pendingPromise.resolve(data.result);
     }
   } else if (ZAF_EVENT.test(data.key)) {
-    var key = data.key.replace(ZAF_EVENT, '');
-    if (client) {
+    var key = data.key.replace(ZAF_EVENT, ''),
+        msg = { appGuid: client._appGuid };
```

Comment:
nitpick: could define msg inside the reply if statement, and define key inline (it's not used in the else block)